frankmarra/DLCM
# Add SVGR and starter icons collection

[SVGR](https://react-svgr.com/docs/next/) makes it easy to import SVG files and render them inline so that we have a little more control over their internal parts. I've included a collection of starter icons from the [Phosphor Icons](https://phosphoricons.com/) library.

### Testing

* Start up the app and go to the homepage. I've added an icon in the `index.js` file which you will see rendered on the page.
* Go to another file and import an icon from the `public/icons/` folder. There is a path alias, so you can type `import Icon[Name] from "@/icons/[icon-name].svg"`. The import name can be called anything you want, but it's good to keep it consistent throughout the app, which is why I prefix it with `Icon`.
